### PR TITLE
moved github oauth secret out of prow

### PR DESCRIPTION
**Pull request by geekodour**
related #208 

Since alertmanager will be using a webhook to connect to GitHub.

the webhook will require the GitHub oauth token when creating its `Deployment` , this PR moves the Github OAuth secret out of the `/prow` directory to `/cluster-infra`

Signed-off-by: Hrishikesh Barman <hrishikeshbman@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ export AUTH_FILE=<path to service-account.json>
 
 ---
 
+- Follow [Getting GitHub OAuth API token](#getting-github-oauth-api-token)
```
Comment (krasi-georgiev) [inline README.md]: Instead of a link just move the paragraph here.

File: README.md [hunk 2]
```diff
@@ -60,7 +62,7 @@ export DOMAIN_NAME=prombench.prometheus.io // Can be set to any other custom dom
 
 ---
 
-- Follow [Setting GitHub API and webhook](#setting-up-github-api-and-webhook-to-trigger-tests-from-comments)
+- Follow [Setting up GitHub webhook](#setting-up-github-webhook-to-trigger-tests-from-comments)
```
Comment (krasi-georgiev) [inline README.md]: we wouldn't need this anymore as the step would be dine when deploying prometheus-meta and grafana.

Comment (geekodour) [reply README.md]: not sure if I understood the change you wanted clearly, but I made some changes based on what i understood. please let me know if I need to make any other changes.

**Comment by krasi-georgiev** [comment]
Actually this change would make sense when we add the Alertmanager details to the readme as otherwise it is not clear why deploying Prometheus and Grafana requires an token to make github requests.

**Comment by geekodour** [comment]
Superseded by #208

**Pull request closed without merging**